### ISSUE TITLE
How do I remove the plugin without launching Intellij?

### ISSUE THREAD

**Issue by alwyn** [git-time-metric/gtm-jetbrains-plugin#4]
Hi there,  I'm on ultimate 16.3.5 and since I've installed gtm and the gtm plugin it appears as if gtm has been hogging some resource to such an extent to paralyze idea.

Edit: I finally got it removed.  PS. I'm using gtm on windows.  It also happens the moment I try to open a file for editing.

**Comment by mschenk42** [comment]
That's really odd but we did recently update the plugin.  I'm wondering if that caused an issue.

Can you confirm gtm is installed from the command line.  Try running`gtm --version`.

@dnedrow any idea on what may be happening here?

**Comment by mschenk42** [comment]
One more question, I don't recognize the version `ultimate 16.3.5`.  What Jetbrain product is that?  I don't see it listed on their site.

**Comment by alwyn** [comment]
Sorry, I meant Intellij Idea Ultimate edition version 2016.3.5.

I have not checked if it is using the same gtm.exe process or if it is continuously spawning new ones, but I have confirmed that the moment GTM kicks into action everything grinds to a halt.

**Comment by alwyn** [comment]
I am using 1.2.6 of the windows build of gtm.  Installed with the installer.

**Comment by alwyn** [comment]
The only other oddball config I have is that my Intellij cache is on ramdisk, but the sources are on SSD which I think should be fine.  The only other subprocess of idea is fsnotifier.

**Comment by mschenk42** [comment]
I'll do some testing over the weekend to see if the plugin updates are causing issues with Windows.  I use MacOS and things seem fine.

GTM itself does very little.  It gets called when certain editor events fire within the IDE.  It creates event files in the `.gtm` directory, which is in the root of your git project.  It's a very simple process for recording time.

**Comment by mschenk42** [comment]
Can you try installing the previous version 1.0.8? 
https://plugins.jetbrains.com/plugin/8494-git-time-metric?pr=idea

**Comment by mschenk42** [comment]
I did some testing this weekend in Windows.  I didn't find any issues with the latest version of the plugin or gtm.  Things are working as expected.  

How are you determining that the high cpu utilization is coming from gtm?  Are you using the Windows task manager? You should not see `gtm.exe` running all the time.  It should run quickly and exit.  It's triggered by editor events like opening, saving and switching tabs.

**Comment by alwyn** [comment]
In my case gtm continiously run using about 13% cpu, but for some reason it is enough to kill Intellij.

When it was happening I would open my ide with no editor tabs open.  Doing Ctrl+N to search for file to open and selecting the file it kicks in the moment I select the file and after that nothing happens and I can do nothing.  So it is not as if I'm changing the file, I don't even get to edit it.

There's no way that indexing could trigger gtm for every file?

Have not tried 1.0.8 yet, but I can if you still think it will give us more info on what's happening.

**Comment by alwyn** [comment]
I have re-enabled the plugin version 1.0.9 and what I experience is that even though I have editor tabs open yet (the ide is still saying loading project opening editors and scanning for files to index), the GTM widget is recording time.  I am now on 14 minutes and I haven't done a thing.

**Comment by alwyn** [comment]
ok that took 36 minutes to finish.  I'm thinking for some reason the scanning is triggering the VisibleAreaChangeListener.

**Comment by mschenk42** [comment]
I'll rollback the previous plugin release and publish a new version tonight.

**Comment by alwyn** [comment]
A also noticed that the pid would stay the same for a couple of minutes and then change.

**Comment by mschenk42** [comment]
That's odd. It should run and exit within milliseconds.

**Comment by alwyn** [comment]
I have downloaded the source and is now running it as a plugin inside the sand-box.  All is ok until I try to open the first file, then it just sits there.  Think it might actually be gtm that's the issue.

Is there a way to turn on debugging for the gtm executable?

**Comment by mschenk42** [comment]
I'd be surprised if it was but anything is possible :).  Try installing the latest version of gtm - version 1.2.7.

**Comment by alwyn** [comment]
yes it could even be my IT department, I try to not always blame them first.

**Comment by mschenk42** [comment]
I just published a new version of the plugin, version 1.0.10.  It should be available via JetBrain's plugin install.  This reverts the changes from the prior release.

**Comment by alwyn** [comment]
ok giving it a try

**Comment by alwyn** [comment]
With 1.2.7 and 1.0.10 I get the same when trying to open a file.  gtm process runs at 13% cpu until its memory allocation reaches around 21MB, then it is replaced by a new gtm process and the cycle continues.

I've seen nothing in my windows event logs, but is it possible that some kind of access denial could cause this?

**Comment by mschenk42** [comment]
I don't know why that is happening for you.  I just finished testing in Windows and I did not run into any issues.  Have you been able run gtm successfully in the past?

One thing you can try is a fresh git project and see what happens.  I'm also wondering if there may be something else interfering like Virus scanning or lack of memory, etc. Just guessing here.

**Comment by krischik** [comment]
To bad no one answered the actual question as I have a problem with `com.maddyhome.idea.color.ColorInfoPluginImpl` so I could really do with information how to “remove the plugin without launching Intellij?”
